src/Razor/Microsoft.AspNetCore.Razor.Language/src/Extensions/IReadOnlyListExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor.Language.Extensions
+{
+    internal static class IReadOnlyListExtensions
+    {
+        public static bool Any<T, TArg>(this IReadOnlyList<T> list, Func<T, TArg, bool> predicate, TArg arg)
```

Comment:
Should sprinkle some null checks on list & predicate for both of these

Reply:
done